progress/progress.go

```diff
@@ -79,6 +79,14 @@ func WithSolidFill(color string) Option {
 	}
 }
 
+// WithCustomFillCharacters sets the characters used to construct the full and empty components of the progress bar.
+func WithCustomFillCharacters(full rune, empty rune) Option {
```

Comment:
Mind changing this to simply `WithFillCharacters`?

Reply:
Done! Thanks for taking a look at this!